unlock-app/src/pages/log.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import UnlockPropTypes from '../propTypes'
+import Layout from '../components/interface/Layout'
+import CreatorAccount from '../components/creator/CreatorAccount'
+import BrowserOnly from '../components/helpers/BrowserOnly'
+import GlobalErrorConsumer from '../components/interface/GlobalErrorConsumer'
+import GlobalErrorProvider from '../utils/GlobalErrorProvider'
+import { pageTitle } from '../constants'
+import configure from '../config'
+
+const { chainExplorerUrlBuilders } = configure()
+
+export const humanize = type => {
+  if (!type) return ''
+  let parts = type.split('_')
+  return parts.map(part => part[0] + part.slice(1).toLowerCase()).join(' ')
+}
+
+export const Log = ({ account, network, transactionFeed }) => {
+  return (
+    <GlobalErrorProvider>
```

Comment:
I could be wrong but I believe the GlobalErrorProvider is passed down from the _app.js or _document.js

Reply:
It isn't but maybe it should be, at the _app.js level would make sense.

Reply:
ok